java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -253,6 +254,39 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         assertNull(data);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void hset_hget_existing_fields_non_existing_fields(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String field1 = UUID.randomUUID().toString();
+        String field2 = UUID.randomUUID().toString();
+        String value = UUID.randomUUID().toString();
+        Map<String, String> fieldValueMap = Map.of(field1, value, field2, value);
+
+        assertEquals(2, client.hset(key, fieldValueMap).get());
+        assertEquals(value, client.hget(key, field1).get());
+        assertEquals(value, client.hget(key, field2).get());
+        assertNull(client.hget(key, "non_existing_field").get());
+    }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void hdel_multiple_existing_fields_non_existing_field_non_existing_key(BaseClient client) {
```

Comment:
```suggestion
    public void hset_hdel_multiple_existing_fields_non_existing_field_non_existing_key(BaseClient client) {
```

Reply:
The functionality is specifically to test Hdel copied over from Node. (Can't test hdel without hset :D)